tests/integration/test_demo_functionality.py

```
@@ -0,0 +1,244 @@
+            _non_string_request = {"text": 123}
+            # This should fail validation in the demo
+            assert False, "Non-string input should be rejected"
+        except (TypeError, ValueError):
+            # Expected behavior
+            pass
+        
+        # Test None input
+        try:
+            _none_request = {"text": None}
+            # This should fail validation in the demo
+            assert False, "None input should be rejected"
```

Ruff (0.12.2)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)